.github/workflows/linux.yml

```diff
@@ -107,7 +107,7 @@ jobs:
         run: |
           # Set the correct version in cmake
           PACKAGE_VERSION=${{ inputs.package_complete_version }}
-          EXTRA_CMAKE_ARGS="-DPERFORMOUS_VERSION=${PACKAGE_VERSION}"
+          EXTRA_CMAKE_ARGS="-DCOMPOSER_VERSION=${PACKAGE_VERSION}"

```

Comment:
I was iffy about this because I'm not sure it's actually implemented in `CmakeLists.txt`, but yeah, good catch, it should be called this.

Reply:
hehe it isnt implemented